provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -31,7 +31,7 @@ namespace Microsoft.Azure.Devices.Provisioning.Client.Transport
     /// </summary>
     public class ProvisioningTransportHandlerMqtt : ProvisioningTransportHandler
     {
-        private static MultithreadEventLoopGroup s_eventLoopGroup = new MultithreadEventLoopGroup();
+        private static readonly MultithreadEventLoopGroup s_eventLoopGroup = new MultithreadEventLoopGroup();
```

Comment:
Would be good to rename this variable. C# naming guidelines tell us not to use underscores.

Reply:
Our style guidelines state to use underscore as a prefix for private fields, and s_ for private static fields.

The mono repo guidelines are the same, actually.